lib/experimental/new_embed.dart

```diff
@@ -462,6 +493,13 @@ class NewEmbed {
       return;
     }
 
+    if (context.dartSource.isEmpty) {
+      dialog.showOk('No code to execute!',
```

Comment:
nit: the `!` seems a little too whimsical here

Reply:
Fair enough. 😄